_games/2025/2025-04-27-kosei.html

```diff
@@ -0,0 +1,33 @@
+---
+date: 2025-04-27
+season: spring
+game_name: 令和7年度 春季都大会 WEEK3
+place: 佼成学園グラウンド
+start_at: "10:30"
+vs: 佼成学園
+vs_full: 佼成学園 LOTUS
```

Comment:
```suggestion
vs_full: 佼成学園高等学校 LOTUS
```

vs_full は対戦校のフルネールを書いているので、学校名もできる限り正式表記をしたいです。

Reply:
指摘通り修正しました